engine/src/main/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationImpl.java

```diff
@@ -750,6 +751,8 @@
 
   protected boolean isExecutionTreePrefetchEnabled = true;
 
+  protected boolean isCompositeIncidentHandlersEnabled = false;
```

Comment:
Let's leave a nice Javadoc here explaining what are the implications of setting this to true, i.e., that multiple incident handlers can be added for the same Incident type, however, only the output/result from the "main" incident handler. 

An example of how to set the main incident handler might be useful as well.

Reply:
Done